plugins/woocommerce/src/Internal/Api/Main.php

```
@@ -27,6 +27,20 @@ class Main {
+	 * Option name for the "Maximum query depth" setting.
+	 *
+	 * Caps how deep the selection tree of a GraphQL query may nest.
+	 */
+	public const OPTION_MAX_QUERY_DEPTH = 'woocommerce_graphql_max_query_depth';
+
+	/**
+	 * Cached result of the feature-enabled check, null until first evaluated.
+	 *
+	 * @var ?bool
+	 */
+	private static ?bool $enabled = null;
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$enabled'. (undefined)

(UnusedPrivateField)